release-notes/10.0/preview/preview1/README.md

```diff
@@ -0,0 +1,36 @@
+# .NET 10 Preview 1 - Release Notes
+
+.NET 10 Preview 1 released on February X, 2025. Find more information on new features released in .NET 10 Preview 1 by browsing through the release notes below:
+
+- [Libraries](./libraries.md)
+- [Runtime](./runtime.md)
+- [SDK](./sdk.md)
+
+## Feature Release Notes

```

Comment:
I don't find this split intuitive.

Reply:
Same... could just remove... technically "workloads"

Reply:
Except EF isn't a workload. It's a mix of stuff.